packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -23,7 +23,6 @@ import {
   disableLegacyContext,
   enableDebugTracing,
   enableSchedulingProfiler,
-  warnAboutDeprecatedLifecycles,
   enableLazyContextPropagation,
 } from 'shared/ReactFeatureFlags';
```

Comment:
So, I'm total noob here. But is it possible that this file somehow/somewhere dynamically generated? And this `warnAboutDeprecatedLifecycles` may be false in some cases?